docs/targets/solana.rst

```diff
@@ -450,6 +450,14 @@ contracts on chain. Examples are available on Solang's integration tests.
 See `system_instruction_example.sol <https://github.com/hyperledger/solang/blob/main/integration/solana/system_instruction_example.sol>`_
 and `system_instruction.spec.ts <https://github.com/hyperledger/solang/blob/main/integration/solana/system_instruction.spec.ts>`_
 
+Minimum balance
++++++++++++++++
+
+In order to instantiate a contract, you need the minimum balance required for a Solana account of a given
+size.  There is a function ``minimum_balance(uint64 space)`` defined in
+`minimum_balance.sol <https://github.com/hyperledger/solang/blob/main/solana-library/minimum_balance.sol>`_
+that calculates this.
```

Comment:
I've re-read this sentence and, although it is correctly written, I thought that `to calculate this` sounds better. This is more style, so only change if you share my opinion.

Reply:
I agree, that sounds better